<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_basic">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Link">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>112</red>
           <green>64</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Link">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>112</red>
           <green>64</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Link">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>112</red>
           <green>64</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="comboBox_axis">
        <item>
         <property name="text">
          <string>X,Y Piezo Stepper</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Z Piezo Stepper</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X,Y Piezo Scanner</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Z Piezo Scanner</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_XY">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_xposition">
           <property name="text">
            <string>X position</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_actualPositionX">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_yposition">
           <property name="text">
            <string>Y position</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_actualPositionY">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_Z">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_zposition">
           <property name="text">
            <string>Z position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_actualPositionZ">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_stop">
        <property name="text">
         <string>STOP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_configurate">
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButton_configurateStepper">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>104</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>configurate Stepper</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_amplXY">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>205</width>
        <height>118</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_amplitudeX">
         <property name="text">
          <string>Amplitude X</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_amplitudeX">
         <property name="suffix">
          <string>V</string>
         </property>
         <property name="maximum">
          <double>999999999999.989990234375000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_frequencyX">
         <property name="text">
          <string>Frequency X</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_frequencyX">
         <property name="suffix">
          <string>Hz</string>
         </property>
         <property name="maximum">
          <double>9999999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_amplitudeY">
         <property name="text">
          <string>Amplitude Y</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_amplitudeY">
         <property name="suffix">
          <string>V</string>
         </property>
         <property name="maximum">
          <double>999999999999.989990234375000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_frequencyY">
         <property name="text">
          <string>Frequency Y</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_frequencyY">
         <property name="suffix">
          <string>Hz</string>
         </property>
         <property name="maximum">
          <double>9999999999.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_amplZ">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>411</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_frequencyY_3">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="suffix">
        <string>Hz</string>
       </property>
       <property name="maximum">
        <double>9999999999.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_amplitudeY_3">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>119</width>
         <height>20</height>
        </rect>
       </property>
       <property name="suffix">
        <string>V</string>
       </property>
       <property name="maximum">
        <double>999999999999.989990234375000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_frequencyY_3">
       <property name="geometry">
        <rect>
         <x>25</x>
         <y>70</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Frequency Y</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_amplitudeY_3">
       <property name="geometry">
        <rect>
         <x>25</x>
         <y>20</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Amplitude Y</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_moving">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBox_kindOfMove">
        <item>
         <property name="text">
          <string>continuous</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>step</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>move absolute</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>move relative</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_distance">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_sortMove">
           <property name="text">
            <string>to absolute position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_distance">
           <property name="maximum">
            <double>10000000000000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_unit">
           <property name="currentText">
            <string>um</string>
           </property>
           <item>
            <property name="text">
             <string>nm</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>um</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>mm</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_info">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="label_speed_stepX">
           <property name="text">
            <string>speed X</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_speedsize_stepsizeX">
           <property name="text">
            <string>unavailable</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_speed_stepY">
           <property name="text">
            <string>speed Y</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_speedsize_stepsizeY">
           <property name="text">
            <string>unavailable</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="groupBox_arrows">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_up">
           <property name="text">
            <string>up</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_left">
           <property name="text">
            <string>left</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_right">
           <property name="text">
            <string>right</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_down">
           <property name="text">
            <string>down</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_scanner">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="label_scannerY">
        <property name="text">
         <string>move scanner Y</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_scannerX">
        <property name="text">
         <string>move scanner X</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_scannerY">
        <property name="suffix">
         <string>V</string>
        </property>
        <property name="maximum">
         <double>9999999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_scannerX">
        <property name="suffix">
         <string>V</string>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
